packages/untp-test-suite/src/interfaces/utils/common.ts

```diff
@@ -1,4 +1,13 @@
 import { readFile } from 'fs/promises';
+import {
+  ICredentialTestResult,
+  IFinalReport,
+  TemplateName,
+  IValidatedCredentials,
+  TestSuiteMessage,
+  FinalStatus,
+} from '../../core/types';
```

Comment:
The `untp-test-suite` is a module package, so I think we need to include '.js' at the end of the file import lines.